config.py

```diff
@@ -0,0 +1,3 @@
+elevator_motor_extend_id = 0
```

Comment:
```suggestion
elevator_extendMotor_id = 0
```
Name config variables with the subsystem_variablename_what-the-config-is-for

Reply:
I used that variable because that's how it was named in their elevator subsystem file to show that's where I was getting the variable from. I meant to only recommend snake case (I clarified this for them in person).